tests/upgrades/test_repository.py

```diff
@@ -164,47 +158,27 @@ def test_pre_scenario_custom_repo_check(self, target_sat):
         content_view = publish_content_view(org=org, repolist=repo)
         content_view.version[0].promote(data={'environment_ids': lce.id})
 
-        result = target_sat.execute(
-            f'ls /var/lib/pulp/published/yum/https/repos/{org.label}/{lce.name}/'
-            f'{content_view.label}/custom/{product.label}/{repo.label}/Packages/b/'
-            f'|grep {RPM1_NAME}'
-        )
-
-        assert result.status == 0
-        assert len(result.stdout) >= 1
-
         subscription = target_sat.api.Subscription(organization=org).search(
             query={'search': f'name={product.name}'}
         )[0]
         ak = target_sat.api.ActivationKey(
             content_view=content_view, organization=org.id, environment=lce
         ).create()
         ak.add_subscriptions(data={'subscription_id': subscription.id})
-
-        rhel7_client = dockerize(ak_name=ak.name, distro='rhel7', org_label=org.label)
-        client_container_id = [value for value in rhel7_client.values()][0]
-        client_container_name = [key for key in rhel7_client.keys()][0]
-
-        host_location_update(
-            client_container_name=client_container_name, logger_obj=logger, loc=loc
-        )
-        status = execute(
-            docker_execute_command,
-            client_container_id,
-            'subscription-manager identity',
-            host=DOCKER_VM,
-        )[DOCKER_VM]
-        assert org.name in status
-        install_or_update_package(client_hostname=client_container_id, package=RPM1_NAME)
+        rhel7_client = Broker(container_host='ubi7:latest', host_class=ContentHost).checkout()
```

Comment:
No, for 2 reasons:
1. The parametrization isnt supported yet in upgrade scenarios because we havent yet identified how could we match/assert the pre-upgrade parameter with post-upgrade parameter in pytest sessions.
2. The fixture should not teardown host on pre-upgrade session completion, it has to be preserved for post-upgrade part and in post upgrade part it can be teardown. If not in post-upgrade, then the CI upgrade scenarios pipeline will run `checkin --all` stage at the end.